client/checkout/classic/upe.js

```diff
@@ -215,9 +215,15 @@ jQuery( function ( $ ) {
 				const { client_secret: clientSecret, id: id } = response;
 				paymentIntentId = id;
 
-				hiddenElementsForUPE.init();
-				const appearance = getAppearance();
-				hiddenElementsForUPE.cleanup();
+				let appearance = getConfig( 'upeAppareance' );
```

Comment:
```suggestion
				let appearance = getConfig( 'upeAppearance' );
```
Pedantic suggestion here, but you know I just couldn't let it slide.

Reply:
Thank you for catching these! Copy and paste strikes again! 😆